Directory.Build.props

```diff
@@ -222,10 +222,16 @@
     <LicenseUrl>https://github.com/dotnet/$(GitHubRepositoryName)/blob/master/LICENSE.TXT</LicenseUrl>
     <RuntimeIdGraphDefinitionFile>$([MSBuild]::NormalizePath('$(LibrariesProjectRoot)', 'Microsoft.NETCore.Platforms', 'pkg', 'runtime.json'))</RuntimeIdGraphDefinitionFile>
     <LicenseFile>$(MSBuildThisFileDirectory)LICENSE.TXT</LicenseFile>
+    <PackageProjectUrl>$(ProjectUrl)</PackageProjectUrl>
+    <PackageIconUrl>http://go.microsoft.com/fwlink/?LinkID=288859</PackageIconUrl>
```

Comment:
I thought PackageIconUrl gave a warning now and instead you are supposed to use an embeded Icon.

[https://docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5048](https://docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5048)

Reply:
Also, Arcade already provides the package icon path which I guess will be used to embed the icon: https://github.com/dotnet/arcade/blob/99b9f805da070d301d76eac578a46dd75d9e5f7c/src/Microsoft.DotNet.Arcade.Sdk/tools/ProjectDefaults.props#L21.